Dockerfile

```diff
@@ -31,10 +31,11 @@ COPY priv ./priv
 COPY qna ./qna
 COPY Makefile ./Makefile
 
-RUN make sbom
+RUN npm ci --prefix ./assets
+RUN npm install -g @cyclonedx/bom@2.0.2
+RUN make sbom_fast
 RUN cp bom* ./assets/static
```

Comment:
This line will fail to run since the sbom files start with `APP_VERSION`  . This is the error I'm getting while building a docker image 
```
cp: cannot stat 'bom*': No such file or directory
```

Reply:
Great catch @TraceyOnim